frontend/controller/service-worker.js

```diff
@@ -68,6 +68,8 @@ const waitUntilSwReady = () => {
     const messageChannel = new MessageChannel()
     messageChannel.port1.onmessage = (event) => {
       if (event.data.type === 'ready') {
+        if (!event.data.currentSyncs) return
```

Comment:
This is a curious line.... could you add a comment about why it's here? Is this to handle old SW versions or something?

Reply:
Yes, this should reasonanly handle old or new SW versions that don't have this field.